### PR TITLE
Add how-to guide for resizing a node pool

### PR DESCRIPTION
**Pull request by jpla-csm**

### CODE REVIEW

**Comment by jpla-csm** [comment]
Hello @dsoitel , 
Je viens d'ajouter la commande "kubectl get nodes" avant le cordon & drain.
JC

File: How-to guides/Resize an AKS node pool.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+**Cordon and drain the nodes**
+
+List all the nodes of the cluster, in order to identify which nodes to delete.
+```bash
+# Connect to AKS context
+kubectl get nodes
+```
+
+Cordon each node of your node pool in order to prevent new pods to get deployed on it.
```
Comment (dsoitel) [inline How-to guides/Resize an AKS node pool.md:74]: ..of the pool +you want to remove+ in order ..

File: How-to guides/Resize an AKS node pool.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+```bash
+# Connect to AKS context
+kubectl cordon "<node_name>"
+```
+
+Drain each node of your node pool in order to remove all pods running.
```
Comment (dsoitel) [inline How-to guides/Resize an AKS node pool.md:80]: ..of the pool +you want to remove+ in order ..

**Review by dsoitel — COMMENTED** [review]
just added a precision in 2 comments

File: How-to guides/Resize an AKS node pool.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                    --max-count 5 \
+                    --min-count 2 \
+                    --enable-cluster-autoscaler \
+                    --labels "cosmotech.com/tier=services" \
+                    --node-taints "vendor=cosmotech:NoSchedule" \
+                    --mode "User" \
+                    --os-type "Linux"
+```
+
+**Cordon and drain the nodes**
```
[the remark this answers is not in the document]
Comment (jpla-csm) [reply How-to guides/Resize an AKS node pool.md:66]: ok fixed

**Comment by jpla-csm** [comment]
Hello @dsoitel , 
modifications faites.

**Comment by dsoitel** [comment]
ok!

**Comment by dsoitel** [comment]
ok

**Review by dsoitel — APPROVED** [review]

**Pull request merged**